actions/generate.action.ts

```diff
@@ -35,7 +35,7 @@ const generateFiles = async (inputs: Input[]) => {
   const spec = inputs.find((option) => option.name === 'spec');
 
   const collection: AbstractCollection = CollectionFactory.create(
-    collectionOption || configuration.collection,
+    configuration.collection||collectionOption ,
```

Comment:
What would happen if there is a collection option passed via CLI, but collection is configured in the `nest-cli.json`? Wouldn't this always take the file config over the CLI config?

Reply:
for now, the collection option always been `@nestjs/schematics` cause it has the default value. file config does not work.